velox/functions/remote/if/GetSerde.cpp

```diff
@@ -27,7 +27,9 @@ std::unique_ptr<VectorSerde> getSerde(const remote::PageFormat& format) {
 
     case remote::PageFormat::SPARK_UNSAFE_ROW:
       return std::make_unique<serializer::spark::UnsafeRowVectorSerde>();
+
+    default:
```

Comment:
looks like this enum only has these two values. Can we remove the default - or add the VELOX_UNREACHABLE() outside of the switch if the compiler still complains?

Reply:
Done, thanks!